app/src/org/commcare/connect/network/ConnectNetworkHelper.java

```diff
@@ -81,7 +82,35 @@ private static ConnectNetworkHelper getInstance() {
 
     private static final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd", Locale.getDefault());
     public static Date parseDate(String dateStr) throws ParseException {
-        return dateFormat.parse(dateStr);
+        Date issueDate=dateFormat.parse(dateStr);
+        return issueDate;
+    }
+
+
+    public static Date convertUTCToLocal(String utcDateString) {
```

Comment:
Proposed change: instead of this method, create a method that will create a local date given a UTC date. Then have ConnectJobRecord.sameDay() convert the UTC dates to local before it checks to see if they're the same day.